stream_alert/rule_processor/rules_engine.py

```diff
@@ -13,11 +13,21 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 '''
+import json
 from collections import namedtuple
 from copy import copy
 
 from stream_alert.rule_processor import LOGGER
 
+DEFAULT_RULE_DESCRIPTION = 'No rule description provided'
+
+RULE_ATTRIBUTES = namedtuple('Rule', ['rule_name',
```

Comment:
namedtuples should follow a naming convention similar to classes (camelcase) and not constants. see: [here](https://stackoverflow.com/questions/36371047/python-naming-convention-namedtuples)